homeassistant/components/notify/slack.py

```diff
@@ -51,7 +51,13 @@ def send_message(self, message="", **kwargs):
         import slacker
 
         channel = kwargs.get('target') or self._default_channel
+        data = kwargs.get('data')
+        if data:
```

Comment:
`attachments` will not be defined when `data` is not defined and that will cause an error. Set `attachments` explicitly to `None`